corehq/apps/reports/views.py

```diff
@@ -1654,18 +1654,11 @@ def close_case_view(request, domain, case_id):
         device_id = __name__ + ".close_case_view"
         form_id = close_case(case_id, domain, request.couch_user, device_id)
         msg = _('''Case {name} has been closed.
-            <a href="javascript:document.getElementById('{html_form_id}').submit();">Undo</a>.
+            <a href="{url}" class="post-link">Undo</a>.
             You can also reopen the case in the future by archiving the last form in the case history.
-            <form id="{html_form_id}" action="{url}" method="POST">
-                <input type="hidden" name="closing_form" value="{xform_id}" />
-                {csrf_inline}
-            </form>
         '''.format(
             name=case.name,
-            html_form_id='undo-close-case',
-            xform_id=form_id,
-            csrf_inline=csrf_inline(request),
```

Comment:
is the csrf token being included in the post request somewhere behind the scenes now?

Reply:
yep, [postGo](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/js/hq_extensions.jquery.js#L5) handles it